src/components/accordion/accordion.style.js

```diff
@@ -48,8 +49,17 @@ const StyledAccordionIcon = styled(Icon)`
 `;
 
 const StyledAccordionHeadingsContainer = styled.div`
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  ${({ hasValidationIcon }) => (
+    hasValidationIcon ? 'grid-template-columns: auto auto;' : 'grid-template-rows: auto auto;'
+  )}
+  
+  ${ValidationIconStyle} {
+    height: 20px;
+    position: relative;
+    top: 2px;
+    z-index: 1000;
```

Comment:
In base-theme.config.js we have a lookup for various zIndex values. Could you move this there, and also check how it renders if composed within a {{FullScreenDialog}}, for example.
https://github.com/Sage/carbon/blob/b998f5ca621a44be743341fc4ead8c71bc03e044/src/style/themes/base/base-theme.config.js#L314

Reply:
Ah that's actually not needed, I meant to remove it so will just delete it